pyproject.toml

```diff
@@ -64,3 +64,7 @@ docs = [
   "myst-parser<3.0.0,>=2.0.0",
   "sphinx-autodoc2<1.0,>=0.5",
 ]
+
+[build-system]
+requires = ["hatchling"]
```

Comment:
what it does ? it it possible to add a comment il toml file to explain the need ?

Reply:
This is the default uv setting, I removed it because I thought it was the cause of a failing job but it was necessary